app/models/UserDatabase.scala

```diff
@@ -0,0 +1,11 @@
+package models
+
+import javax.inject.Inject
+
+@javax.inject.Singleton
+class UserDatabase @Inject()() {
+  def lookupUser(u: User): Boolean = {
+    //TODO query your database here
+    if (u.username == "foo" && u.password == "foo") true else false
```

Comment:
don't use literal values like `"foo"` they should come from constants file

Reply:
I've removed this file as it was accidentally pushed to my branch.